machine_management/index.adoc

```diff
@@ -6,54 +6,79 @@ include::_attributes/common-attributes.adoc[]
 
-Machine management is implemented as a xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[Custom Resource Definition](CRD).
+Machine management is implemented as a xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[custom resource definition] (CRD).
 A CRD object defines a new unique object `Kind` in the cluster and enables the Kubernetes API server to handle the object's entire lifecycle.
 
 The Machine API Operator provisions the following resources:
 
-* MachineSet
-* Machine
-* Cluster Autoscaler
-* Machine Autoscaler
-* Machine Health Checks
+* `MachineSet`
+* `Machine`
+* `ClusterAutoscaler`
+* `MachineAutoscaler`
+* `MachineHealthCheck`
 
 [discrete]
 == What you can do with compute machine sets
 
 As a cluster administrator you can:
 
 * Create a compute machine set on:
+
 ** xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-aws[AWS]
+
 ** xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#creating-machineset-azure[Azure]
+
 ** xref:../machine_management/creating_machinesets/creating-machineset-gcp.adoc#creating-machineset-gcp[GCP]
-** xref:../machine_management/creating_machinesets/creating-machineset-osp.adoc#creating-machineset-osp[OpenStack]
-** xref:../machine_management/creating_machinesets/creating-machineset-rhv.adoc#creating-machineset-rhv[RHV]
+
+** xref:../machine_management/creating_machinesets/creating-machineset-osp.adoc#creating-machineset-osp[{rh-openstack}]
+
+** xref:../machine_management/creating_machinesets/creating-machineset-rhv.adoc#creating-machineset-rhv[{rh-virtualization}]
+
 ** xref:../machine_management/creating_machinesets/creating-machineset-vsphere.adoc#creating-machineset-vsphere[vSphere]
 
 * xref:../machine_management/manually-scaling-machineset.adoc#manually-scaling-machineset[Manually scale a compute machine set] by adding or removing a machine from the compute machine set.
-* xref:../machine_management/modifying-machineset.adoc#modifying-machineset[Modify a compute machine set] through the MachineSet YAML configuration file.
+
+* xref:../machine_management/modifying-machineset.adoc#modifying-machineset[Modify a compute machine set] through the `MachineSet` YAML configuration file.
+
 * xref:../machine_management/deleting-machine.adoc#deleting-machine[Delete] a machine.
+
 * xref:../machine_management/creating-infrastructure-machinesets.adoc#creating-infrastructure-machinesets[Create infrastructure compute machine sets].
+
 * Configure and deploy a xref:../machine_management/deploying-machine-health-checks.adoc#deploying-machine-health-checks[machine health check] to automatically fix damaged machines in a machine pool.
 
+[discrete]
+== What you can do with control plane machine sets
+
+As a cluster administrator you can:
```

Comment:
See the [ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) on lead-in wording for lists. This would also apply to lines 28 and 30 above. 
```suggestion
As a cluster administrator, you can perform the following actions:
```
In the guidance, you can also use _these_ instead of _the following_.

Reply:
Perhaps as something to take up as a continuous improvment task, but part of the ISG guidance on lists is that it is important to avoid having a lot of lists as the main content. It can lead to documentation that looks more like an outline than complete work.